courses/views.py

```diff
@@ -128,6 +128,7 @@ def create_ccx(request):
         user_email (string): email address of the user to make CCX Coach
         display_name (string): Title of the CCX
         max_students_allowed (int): Maximum number of students to allow into the ccx
```

Comment:
yes, but on line 156, we expect it to be `total_seats` when coming from the `POST`.